examples/custom-middleware/tenant-group-manipulation.md

```diff
@@ -0,0 +1,46 @@
+---
+description: Tenant group manipulation
+---
+
+# Tenant group manipulation
+
+We can change a default tenant and order of the tenants in the ROR menu by providing `defaultGroup` query string parameter before the login, and chenge the order of `availableGroups`
```

Comment:
"We can change the default tenancy, and the display ordering of the tenancies in the ROR menu by providing the `defaultGroup` query parameter in the HTTP request submitted by the login form."

Reply:
Done ✔️